README.md

```diff
@@ -4,46 +4,55 @@
 
 This is a static site generated with Middleman, using [alphagov/tech-docs-template](https://github.com/alphagov/tech-docs-template).
 
-Some of the files (like the CSS, javascripts and layouts) are managed in the template and are not supposed to be modified here. Any project-specific
-Ruby code needs to go into `/app`.
+Some of the files (like the CSS, javascripts and layouts) are managed in the template and are not supposed to be modified here. Any project-specific Ruby code needs to go into `/app`.
 
-## Build the app locally
+## Run the app locally
 
-```sh
-bundle install
-```
+Run govuk-developer-docs either inside or outside govuk-docker, configuring its behaviour with the ENV variables below.
 
-## Run the tests locally
+### ENV variables
 
-```
-bundle exec rake
-```
+- `GOVUK_TOKEN=<your private token>` - token to use to make authenticated requests to GitHub's API. Authenticated requests have a much higher rate limit. *You _will_ need to specify a `GITHUB_TOKEN` if you want to build the entire Developer Docs site*. [Create the token on GitHub](https://github.com/settings/tokens/new) (the token doesn't need any scopes).
+- `SKIP_PROXY_PAGES=true` - avoid fetching remote 'docs/' for each repo (i.e. just build the docs that live within govuk-developer-docs itself). You can use this if you don't have a `GITHUB_TOKEN` or if you don't care about including the remote docs.
+- `NO_CONTRACTS=true` - [recommended setting](https://github.com/alphagov/govuk-developer-docs/commit/4b624a72761490c8e9b99a1aa7a10371415381e6) for speeding up the site build process
 
-## Run the app locally
+### Run with govuk-docker
 
-```sh
-SKIP_PROXY_PAGES=true ./startup.sh
+govuk-docker doesn't have great support for passing ENV vars into the application startup. You'll need to [edit the docker-compose.yml](https://github.com/alphagov/govuk-docker/blob/ed98d3547708286f534598c78fb5c57ee3c8d112/projects/govuk-developer-docs/docker-compose.yml#L12) to add the necessary ENV vars (e.g. `GITHUB_TOKEN`) under the `environment` property for both the `govuk-developer-docs-app` and `govuk-developer-docs-lite` groups, eg:
+
+```yml
+   environment:
+      LANG: "C.UTF-8"
+      GITHUB_TOKEN: "<fill it in>"
```

Comment:
This block is under `govuk-developer-docs-lite`, but when we run `govuk-docker-up` it attaches to the `govuk-developer-docs-app` service, which is the block below. Should we put it in that block instead? or maybe in the common `govuk-developer-docs` above?

Reply:
I've now tweaked the doc to make it clearer to add it to both services. 👍 
I don't think it's possible to add to the common `govuk-developer-docs` because the `environment` property in `-lite` and `-app` _override_ that value rather than merge with it.